Create sidecar mode for ory oathkeeper (#98)
ory/k8s: docs/helm/oathkeeper.md

    --set 'maester.enabled=false' \
    ory/oathkeeper
```

#### Operation modes

> The Maester controller is now available in two modes: 

Reviewer: ```suggestion
The Oathkeeper Maester works in either of these two modes: 
```

**Controller mode**

> In this mode, the controller is a dedicated deployment and scales independently from the Oathkeeper application. All communication with Oathkeeper is based on a configMap object, which stores the Oathkeeper configuration that has been constructed from Oathkeeper-maesters Custom Resource. 

Reviewer: ```suggestion
In this mode, the controller is a dedicated deployment and scales independently from the Oathkeeper application. All communication with Oathkeeper is based on a configMap object, which stores the Oathkeeper configuration created based on the APIRule Custom Resource. 
```

> This mode requires giving elevated privileges to the Oathkeeper-maester controller to allow operations on the configMaps. 

Reviewer: ```suggestion
This mode requires giving elevated privileges to the Oathkeeper Maestercontroller to allow operations on the configMaps. 
```

**Sidecar mode**

> In this mode, the Hydra Maester controller runs as an additional container in the Oathkeeper application Pod. All communication is done on the local filesystem (which can be a shared tempfs, mounted directory or persistent volume), and the controller is scaled together with the Oathkeeper application. 

Reviewer: ```suggestion
In this mode, the Hydra Maester controller runs as an additional container in the Oathkeeper application Pod. All communication is done on the local filesystem, which can be a shared `tempfs`, mounted directory or a persistent volume, and the controller is scaled together with the Oathkeeper application. 
```

    --set 'maester.enabled=false' \
    ory/oathkeeper
```

#### Operation modes
The Maester controller is now available in two modes: 
- global controller
- dedicated sidecar

**Controller mode**

> In this mode, the controller is a dedicated deployment, and scales separately of the oathkeeper application. All communication with oathkeeper is based on a configMap object, which stores the translated oathkeeper configuration. 

Reviewer: ```suggestion
In this mode, the controller is a dedicated deployment and scales independently from the Oathkeeper application. All communication with Oathkeeper is based on a configMap object, which stores the translated Oathkeeper configuration. 
```

`which stores the translated` -> this is not exactly clear to me. I'm guessing it's translated from "oathkeeperish" to "Kuberneres"? Can you rephrase it and make it clearer pls?

> This mode requires elevated privileges for the maester controller in order to operate on the configmaps. 

Reviewer: ```suggestion
This mode requires giving elevated privileges to the Hydra Maester controller to allow operations on the configMaps. 
```

**Sidecar mode**

> In this mode the controller is an extra container running in the oathkeeper pod. All communication is done on local files, and the controller is scaled together with oathkeeper. 

Reviewer: ```suggestion
In this mode, the Hydra Maester controller runs as an additional container in the Oathkeeper application Pod. All communication is done on local files, and the controller is scaled together with the Oathkeeper application. 
```

`All communication is done on local files,` -> not clear. What local files? Local to Oathkeeper? It's Pod? The OS local files?
